### ISSUE TITLE
Node 0.12.7 - Cannot read property 'emit' of null

### ISSUE THREAD

**Issue by nhunzaker** [algolia/faux-jax#15]
I'm testing against multiple versions of node for a project and node 0.12.7 fails for the following reason:

`TypeError: Cannot read property 'emit' of null`

Looks like this occurs in `FakeRequest::respond` around like 189:

```
  function end() {
    res.end();
    res.socket.emit('end');
  }
```

I'm happy to dig into this and resolve the issue - but do you have any immediate thoughts?

**Comment by vvo** [comment]
I think I did a recent commit to fix that

**Comment by vvo** [comment]
Have a look at master, may need to release though

**Comment by nhunzaker** [comment]
Ah, yep! Any idea on a timeframe for a release?

**Comment by vvo** [comment]
In a couple of hours

**Comment by nhunzaker** [comment]
Cool. Pulling straight from Github works great for me until then. Thank you!

**Comment by vvo** [comment]
released 5.0.0

**Comment by nhunzaker** [comment]
Thanks!
